docker-compose.yml

```diff
@@ -5,23 +5,33 @@ services:
     links:
       - db
     ports:
-      - "3000:3000"
-      - "80:80"
+      - '3000:3000'
+      - '80:80'
     volumes:
       - gems:/usr/local/rvm/gems
     environment:
-      RAILS_ENV: 'production'
-      PASSENGER_APP_ENV: 'production'
-      DB_DATABASE: 'fat_free_crm_production'
+      RAILS_ENV: 'development'
```

Comment:
if we're going to have this setup for development, can you rename this file to docker-compose-dev.yml

you can keep the production docker-compose file too